sop-website/docs/Data-Upload-Update/Data-Uploads.mdx

```diff
@@ -0,0 +1,418 @@
+
+### Marking files as deleted
+(Not applicable to Azure Data Share) In `full` mode, files removed from the folder structure will be considered `deleted`.  In `append` mode, a DGS user can manually supply a list of files to be marked as deleted.  
+
+### OMOP EHR Folder
+This folder shall include structured and unstructured 13 OMOP clinical data files(exluding specimen table) as found here (https://ohdsi.github.io/CommonDataModel/cdm54.html#Clinical_Data_Tables). We are not requiring actual notes to be included in the "note_text" field of the NOTE table, but since working with the NOTE_NLP table would greatly benefit from access to foreign keys only present in the NOTE table. Thus the note table should at least contain core identifiers (note_id), descriptors (note_type_concept_id and note_class_concept_id) as well as there foreign keys to PERSON, VISIT_OCURRENCE, VISIT_DETAILS. Each of these table will be in the form of comma delimited files with header rows.
+
+Each OMOP file should container records for ALL ACTIVE patients submitted to CHoRUS to date. The OMOP files is a complete snapshot of current DGS local CHoRUS OMOP tables.  
+
+### Images Folder
+The `Images` folder should contain all images for the patient, with images organized in standard DICOM hierarchy with study/series folders.
+
+File names should follow the format below:
+
+* Patient Identification: Typically includes a person ID.
+* Study Date: The date when the study was conducted, usually in the format YYYYMMDD.
+* Study Time: The time of the study, often in HHMMSS format.
+* Modality: Refers to the type of equipment used for the scan, such as MR (Magnetic Resonance), CT (Computed Tomography), or US (Ultrasound).
+* Series Number: Indicates the sequence of a particular series of images in a study.
+* Instance Number: Represents the specific image number within a series.
+
+Example of a DICOM Image Name
+```
+PersonID_20230101_101530_CT_01_001.dcm
+```
+
+This example would represent a CT scan for a patient, conducted on January 1, 2023, at 10:15:30. This image is the first in its instances and the first in that series. Make sure instance and series match with dicom metadata.
+
+### Waveforms Folder
+
+The `Waveforms` folder should contain all waveforms for the patient organized into `session` folders, each session containing a set of files corresponding to a continuous recording session. 
+
+Waveform data refers to all data acquired from bedside monitors and devices, including alarms, numerics data obtain at regular of irregular intervals, and high-frequency (>1 Hz) waveform data such as digitized EKG tracings. The waveforms should be deidentified.
+
+Files should be named using the following format:
+
+* Patient Identification: A unique person ID, typically a number or numeric digits from OMOP Person Table.
+* Start Date: The date when the waveform data was recorded, in the format YYYYMMDD.
```

Comment:
At the initial waveform SOP meetings in July, we agreed that we would not use date and time in the folder / file naming. Instead we decided to only use `person_id` and `session`. I strongly prefer this approach. It avoids having to time-shift folder and filenames. The current Waveform SOP suggests using `session` instead of date and time. 

We will need to decide how to handle this and should try to be consistent between waveforms and images.

Reply:
Changed based on our discussion.